draft-ietf-quic-invariants.md

```diff
@@ -138,9 +138,9 @@ Long Header Packet {
   Version-Specific Bits (7),
   Version (32),
   DCID Len (8),
-  Destination Connection ID (0..160),
+  Destination Connection ID (0..255),
```

Comment:
Yikes, we both got this wrong.  (And then I got it wrong again by putting 2047 here first.)

```suggestion
  Destination Connection ID (0..2040),
```